pkgs/top-level/all-packages.nix

```diff
@@ -22925,6 +22925,8 @@ with pkgs;
   mistralclient = with python311Packages; toPythonApplication python-mistralclient;
   swiftclient = with python311Packages; toPythonApplication python-swiftclient;
   troveclient = with python311Packages; toPythonApplication python-troveclient;
+  watcherclient = with python311Packages; toPythonApplication python-watcherclient;
+  zunclient = with python311Packages; toPythonApplication python-zunclient;
```

Comment:
Don't we need to add `zaqarclient` here?

Reply:
Yes I didn't add zaqarclient client here, because python-zaqarclient doesn't provide any console_scripts unfortunately.

Source : https://github.com/openstack/python-zaqarclient/blob/master/setup.cfg#L32